webpack.config.js

```diff
@@ -7,7 +7,7 @@ var HtmlWebpackPlugin = require('html-webpack-plugin');
 
 var externalAssets = {
   css: [
-    'https://assets.zendesk.com/apps/sdk-assets/css/2/zendesk_garden.css'
+    'https://cdn.jsdelivr.net/combine/npm/@zendeskgarden/css-bedrock@6,npm/@zendeskgarden/css-arrows@2,npm/@zendeskgarden/css-avatars@2,npm/@zendeskgarden/css-buttons@5,npm/@zendeskgarden/css-callouts@2,npm/@zendeskgarden/css-chrome@2,npm/@zendeskgarden/css-forms@5,npm/@zendeskgarden/css-menus@6,npm/@zendeskgarden/css-modals@5,npm/@zendeskgarden/css-pagination@2,npm/@zendeskgarden/css-tables@2,npm/@zendeskgarden/css-tabs@4,npm/@zendeskgarden/css-tags@3,npm/@zendeskgarden/css-tooltips@3,npm/@zendeskgarden/css-utilities@2'
```

Comment:
May make sense to generate this URL based on an object of package and version key-value pairs, so devs can more easily pick and choose what they need.

Reply:
Yea, I would much prefer a `zendeskGardenAssets` hash with package and version key/value, like:

```
zendeskGardenAssets: {
  'css-tabs': 4
}
``` 

It does feel a little like rebuilding dependency management though.  Also, jsdelivr resolves @4 to a major version, yea?

Reply:
You could just specify them in package.json as real dependencies and then read that in your webpack config. That way you get nice tooling around updating/removing packages.

Reply:
Something like:

```js
const { dependencies } from 'package.json';

const pkg = Object.keys(dependencies).filter(item => 
item.includes('@zendeskgarden'));

const url = pkg.reduce((url, pkg) => {
  return url = `${url}${pkg}@${dependencies[pkg].replace(/^[\^~]/g,'')},`
}, "https://cdn.jsdelivr.net/combine/npm/").slice(0, -1);
```

Reply:
Uhh nice!

Reply:
It's a nice approach to use package.json semantics to include the packages.  I feel a little bit like having deps. that aren't actually included in the build is sort of redundant.  Perhaps they could be described as devDependencies, so that they wouldn't be included in any prod build?

Reply:
Looks good! Let's not include `chrome` in here – can't think of any good reason why app developers should be using that.

Reply:
@svizzari yeah devDeps can work if you're worried about it installing when they do `npm install`. I don't think that's a problem they're using the deps and managing the versions and assets in this build it's just that when they produce their app it generates the url instead of bundling the assets.

Reply:
Sure thing, I don't see any problem provided we configure the webpack build
to ensure the garden assets won't automatically get bundled into the final
assets

Reply:
The only problem I see with using package.json is that we won't be able to distinguish between css and js 🤔

Reply:
You can regexp the name `/@zendeskgarden\/css/.test(package)`